<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contacts</class>
 <widget class="QDialog" name="contacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>565</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>741</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>745</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>联系人</string>
  </property>
  <property name="windowIcon">
   <iconset resource="souce.qrc">
    <normaloff>:/souce/souce/登录.png</normaloff>:/souce/souce/登录.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="contactsBack" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;通讯录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="contactsBox" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>1000</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219,219,219);
</string>
    </property>
    <widget class="QWidget" name="contactINfo" native="true">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>0</y>
       <width>521</width>
       <height>550</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(219,219,219);</string>
     </property>
    </widget>
    <widget class="QListWidget" name="contactsList">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>10</y>
       <width>221</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#contactsList{border:none;background-color: rgb(238, 238, 238);}


#contactsList::Item{
width:260px;
height:55px;
}
#contactsList::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>55</width>
       <height>55</height>
      </size>
     </property>
     <property name="gridSize">
      <size>
       <width>200</width>
       <height>65</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="createContact">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>26</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>新建联系人</string>
     </property>
     <property name="whatsThis">
      <string>新建联系人</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#createContact{
background-color: rgb(37, 172, 125);
border-radius:13px;
border:none;
font: 75 25px &quot;Consolas&quot;;
color:white;
}
#createContact:hover{
background-color: rgb(37, 172, 125);
	background-color: rgb(46, 213, 155);
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="contInfoTop">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>501</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 238);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="contLogo">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>19</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="contWriteLetter">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contWriteLetter{
	border:none;
	font-size:14px;
	background-color:rgb(219,219,219);
}
#contWriteLetter:hover{
background-color: rgb(26, 227, 159);
color:white;
}</string>
      </property>
      <property name="text">
       <string>写信</string>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/发送.png</normaloff>:/souce/souce/发送.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="delContact">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#delContact{
	border:none;
	font-size:14px;
	background-color: rgb(219,219,219);
}
#delContact:hover{
background-color:tomato;
color:white;
}</string>
      </property>
      <property name="text">
       <string>删除联系人</string>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/关闭1.png</normaloff>:/souce/souce/关闭1.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>43</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>66</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>邮   箱:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contName">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>44</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contName{border:none;}
#contName:hover{
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="contEmail">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>72</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contEmail{
border:none;
}
#contEmail:hover{
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="createAlert">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:tomato;</string>
      </property>
      <property name="text">
       <string>请在此添加用户信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="must2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:tomato;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QLabel" name="must1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:tomato;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="contInfoBottom">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>501</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 238);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>18</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>基本信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>68</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>备注：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>98</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>联系电话:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>社交信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>QQ:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>WeChat:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contRemarkName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>67</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contPhone">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>98</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contQQ">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contWechat">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="contReset">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contReset{
	border:none;
	font-size:14px;
color:white;
	background-color: rgb(29, 171, 123);
}
#contReset:hover{
background-color: rgb(26, 227, 159);
}</string>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="contSave">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>310</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contSave{
	border:none;
	font-size:14px;
color:white;
	background-color: rgb(29, 171, 123);
}
#contSave:hover{
background-color: rgb(26, 227, 159);
}</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="sync">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>470</y>
       <width>26</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>同步联系人</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#sync{
background-color: rgb(37, 172, 125);
border-radius:13px;
border:none;
font: 75 25px &quot;Consolas&quot;;
color:white;
}
#sync:hover{
background-color: rgb(37, 172, 125);
	background-color: rgb(46, 213, 155);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="souce.qrc">
       <normaloff>:/souce/souce/同步.png</normaloff>:/souce/souce/同步.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="contClose">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#contClose{
	background-color:tomato;
	border-radius:10px;}
#contClose:hover{border-image: url(:/souce/souce/关闭1.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="contMin">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">#contMin{
background-color:rgb(255,224,81);
	border-radius:10px;}
#contMin:hover{
	border-image: url(:/souce/souce/最小化.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="contUserMail">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>211</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#contUserMail{
	font: 75 12px &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="contUserLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>contactsBox</zorder>
   <zorder>contactsBox</zorder>
   <zorder>contClose</zorder>
   <zorder>contMin</zorder>
   <zorder>contUserMail</zorder>
   <zorder>contUserLogo</zorder>
  </widget>
 </widget>
 <resources>
  <include location="souce.qrc"/>
  <include location="avatars.qrc"/>
 </resources>
 <connections/>
</ui>
